src/views/TheWalletView.vue

```diff
@@ -89,6 +89,7 @@ export default {
     },
     network() {
       this.web3.eth.clearSubscriptions();
+      if (this.online && !this.isOfflineApp) this.setup();
```

Comment:
move this change to the web3 watcher since web3 will change if the network changes.